iceoryx_utils/platform/linux/include/iceoryx_utils/platform/acl.hpp

```diff
@@ -11,7 +11,9 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-#pragma once
+#ifndef IOX_UTILS_PLATFORM_ACL_HPP
```

Comment:
this is not completely unique, since the QNX, windows and mac platform header will have the same include guard. It shouldn't be a problem, though, since they should never be used in parallel.

What do the other guys think? Should we keep it this way or maybe add the platform name to the guard?

Reply:
Should be unique to avoid accidential mixups. Would most likely not compile if e.g. windows and mac headers were mixed.

Reply:
I had this conversation with @elfenpiff before I modified it. His take was that these includes are separated in their include paths on cmake level and therefore cannot mix.
@elfenpiff , @budrus please comment and it'll be modified if necessary.

Reply:
Added platform name to guard.